README.md

```diff
@@ -4,106 +4,6 @@
 
 QIIME2 plugin for paired sample comparisons.
 
-q2-longitudinal's actions support statistical and visual comparisons of longitudinal study designs and paired samples, to determine if/how samples change between observational "states". "States" will most commonly be related to time, and the sample pairs should typically consist of the same individual subject  observed at two different time points. For example, patients in a clinical study whose stool samples are collected before and after receiving treatment.
+Use examples are provided in the [QIIME2 tutorial documentation](https://docs.qiime2.org/2017.8/tutorials/longitudinal/).
```

Comment:
We probably don't want to hard-link to a versioned set of the docs here, otherwise we will either be on the hook for keeping this up-to-date (a pain in the monthly release cycle era), or users in the future will be greeted with an "outdated docs" warning page. How about we copy one of the [existing READMEs](https://github.com/qiime2/q2-feature-classifier/blob/master/README.md)? There are a few variations of this around here, but I think this is the most common format. Thoughts?